### DISCUSSION TITLE
Error on Startup

### DISCUSSION THREAD

**Discussion by wilarN** [Pryaxis/TShock#2179] [Q&A] [answered]
Hello, I just installed tshock and when im trying to startup the server it's throwing an error. And im not really sure why. // Thanks William
(Also im sorry if this is common and im just not aware of it)

![image](https://user-images.githubusercontent.com/68875195/101287650-d1b6db80-37f1-11eb-9ff8-929307ee4e8c.png)

**Comment by Zaicon** [comment, marked as answer]
It looks like there is an issue with your config file - if this is a new setup, try deleting the config file (tshock/config.json) and letting tShock generate a new one.
